addons/hr_holidays/tests/test_allocations.py

```diff
@@ -2,6 +2,7 @@
 
 from freezegun import freeze_time
 
+from odoo.fields import Command
```

Comment:
Nitpicking but as this is R&D PR, can you move it one line below to keep alphabetical order?